cmd/kubernetes_version.go

```diff
@@ -69,6 +72,30 @@ func extractVersion(data []byte) (string, error) {
 	return ver, nil
 }
 
+func getWebDataRetry(srvURLs []string, token string, cacert *tls.Certificate, timeoutDuration, waitDuration time.Duration) (data []byte, err error) {
+	timeout := time.After(timeoutDuration)
+	wait := time.Duration(waitDuration)
+	for {
+		select {
+		case <-timeout:
+			err = errors.New("timeout - getting Kubernetes version using REST API")
+			return
+		default:
+			for _, srvURL := range srvURLs {
+				data, err = getWebData(srvURL, token, cacert)
+				if err == nil {
+					return
+				}
+				glog.V(2).Info(fmt.Sprintf("HTTP ERROR: %v\n", err))
+				continue
```

Comment:
given that you're at the end of the loop, what is `continue` intended to do here?